doc/bridge_packages/flame_rive/rive.md

```diff
@@ -0,0 +1,44 @@
+# flame_rive
+
+`flame_rive` is a bridge library for using [rive](https://rive.app/) animations in your
+Flame game. Rive is a real-time interactive design and animation tool. you can use it to create
+animation
```

Comment:
Remember that 100 is the max line length and we should only create a newline when it overflows.
```suggestion
`flame_rive` is a bridge library for using [rive](https://rive.app/) animations in your Flame game.
Rive is a real-time interactive design and animation tool. you can use it to create animation
```
Also The last sentence is not finished, and does not start capitalized.

Reply:
![Screen Shot 2022-10-24 at 7 29 36 PM](https://user-images.githubusercontent.com/36990991/197544188-8eb56b74-af55-4607-83de-f6d62d68fb6f.png)

in the editor, it shows there is no line break but when I open changes in the browser they are not the same as in the editor, the bar on the right side is at 100 char,

not sure why they look different